Storage/tests/Unit/Connection/RestTest.php

```diff
@@ -426,47 +421,38 @@ public function validationMethod()
             [
                 ['validate' => true],
                 false,
-                false,
                 'md5'
             ], [
                 ['validate' => true],
                 true,
-                false,
                 'crc32'
             ], [
                 ['validate' => true],
                 false,
-                true,
-                'crc32'
+                'md5'
```

Comment:
This is a change from previous behavior when an extension might be present (& needed for CRC32) so we used to default to md5 only if extension is not present.
Not that we don't need the extension anymore, so this check is not needed and this function will always result in md5 if not explicitly provided with crc32.

Reply:
Since I added the check for availability of crc32c algo, this change in behavior isn't needed anymore. Reverted.